recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -8,27 +8,123 @@ inherit cmake pythonnative
 S = "${WORKDIR}/git"
 OECMAKE_SOURCEPATH = "${S}/"
 
+SUMMARY = "prplMesh"
 SECTION = "net"
 LICENSE = "BSD-2-Clause-Patent.txt"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
-DEPENDS = "python-native python-pyyaml-native json-c openssl zeromq"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=0518d409dae93098cca8dfa932f3ab1b"
+
+DEPENDS = "python-native python-pyyaml-native json-c openssl"
 RDEPENDS_${PN} = "iproute2 busybox readline"
```

Comment:
`RDEPENDS_${PN}` used for runtime dependencies. In Yocto-community this variable mostly used for dependencies from plugin to main application or for dependency between applications (e.g. called with "system()" call). It's uncommon to add shared library to RDEPENDS explicitely, because all required shared libraries can be found with `shdeps` mechanism

Reply:
Done